@navikt/ds-css/grid/container.css

```diff
@@ -0,0 +1,7 @@
+.navds-grid-container {
```

Comment:
Hvor tett er egentlig `navds-grid-container` koblet med griden? Kan den brukes uten grid, og kan grid brukes uten container?

Reply:
Ja, slik det er satt opp nå kan både grid brukes uten container og container kan brukes uten grid.
Men jeg mener fortsatt vi burde ha en <Container /> komponent for å sette en default sidebredde. 
Jeg tester noen komponenter for layout, så mulig vi kan dra den inn der i stedet. Tanker @andnorda?

Reply:
Jeg ser for meg at `container` kan være nyttig også for team som lager sider som ikke passer en av våre foreslåtte layouts, og som heller ikke følger griden helt. Å holde den helt uavhengig vil jo gi mest fleksibilitet, sånn sett. I så fall er kanskje en navneendring på sin plass? Jeg foreslår `.navds-page-container` eller kanskje enda bedre: `.navds-content-container`.